central/deployment/views/container_images.go

```diff
@@ -0,0 +1,28 @@
+package views
+
+// ContainerImagesResponse represents a container image with its associated cluster IDs.
+// This view is used to get distinct container images from active deployments
+// along with the clusters where they are deployed.
+type ContainerImagesResponse struct {
+	ImageIDV2 string `db:"image_id"`
+	// Each imageIDV2 will map to a single digest, but SQL wants us to apply an aggreate func to all selected fields
+	// when grouing by imageIDV2. So we need to select the digest with a min aggregate,
+	// which will be a no-op for a single value.
+	ImageDigest string   `db:"image_sha_min"`
```

Comment:
Would adding the digest to the `group by` effectively accomplish the same thing?